server/supernova/sc/sc_osc_handler.hpp

```diff
@@ -244,6 +244,7 @@ class sc_osc_handler : private detail::network_thread, public sc_notify_observer
         sc_osc_handler* osc_handler;
         boost::endian::big_int32_t msg_size_;
         std::vector<char> msg_buffer_;
+        std::mutex tcp_mutex;
```

Comment:
can you rename this to `socket_mutex_`? indicates the relationship more clearly and the trailing underscore is more consistent with the rest of supernova (i think `osc_handler`'s naming is a typo)

Reply:
Are you sure?
See: https://github.com/supercollider/supercollider/blob/develop/server/supernova/sc/sc_osc_handler.hpp#L126

Reply:
sure, it's not consistent, whatever you think is good, but the variable name would be clearer as `socket_mutex` with or without the underscore

Reply:
done